layouts/default.vue

```diff
@@ -3,7 +3,7 @@
     <nuxt />
     <footer class="footer">
       <div class="content has-text-centered">
-        <p>Imprimis™ footer</p>
+        <p>Imprimis™</p>
```

Comment:
Is Imprimis trademarked? Don't mark things that aren't trademarked as trademarked.

Also if you do use this character, use its HTML escape code notation: `&trade;`. Avoids encoding issues with weird or old browsers.

Reply:
The TM was already there, to be decided at a later time. Perhaps something along the lines of :copyright: {{currentYear}} Team Primis / Imprimis / Libprimis / whoever

Reply:
whomever*